supportedBackends/jes/src/main/scala/cromwell/backend/impl/jes/JesInitializationActor.scala

```diff
@@ -52,22 +61,41 @@ class JesInitializationActor(override val workflowDescriptor: BackendWorkflowDes
   /**
     * Abort all initializations.
     */
+  // TODO PBE ??? is not cool
   override def abortInitialization(): Unit = ???
```

Comment:
Since we believe abort to be feature complete, could you file a quick ticket on this?